python/cudf_polars/docs/overview.md

```diff
@@ -198,6 +243,123 @@ To simplify state tracking, all columns should be considered immutable
 on construction. This matches the "functional" description coming from
 the logical plan in any case, so is reasonably natural.
 
+## Traversing and transforming nodes
+
+As well as just representing and evaluating nodes. We also provide
+facilities for traversing a tree of nodes and defining transformation
+rules in `dsl/traversal.py`. The simplest is `traversal`, this yields
+all _unique_ nodes in an expression parent before child, children
+in-order left to right (i.e. a pre-order traversal). Use this if you
```

Comment:
This sentence is hard form me to read/understand (may just be me).

Reply:
Reworded.